src/app/pages/Explore/Tokens.tsx

```diff
@@ -239,24 +239,26 @@ const ListItem = memo<ListItemProps>(({ assetSlug, last, active, accountPkh }) =
     return undefined;
   }, [displayed, setDisplayed]);
 
-  const renderBalance = useCallback(
+  const renderBalancInToken = useCallback(
     (balance: BigNumber) => (
-      <div className="flex items-center">
-        <span className="text-base font-normal text-gray-700">
-          <Money>{balance}</Money>{' '}
-          <span className="opacity-90" style={{ fontSize: '0.75em' }}>
-            {getAssetSymbol(metadata)}
-          </span>
-        </span>
-
-        <InUSD assetSlug={assetSlug} volume={balance}>
-          {usdBalance => <div className={classNames('ml-2', 'text-sm font-light text-gray-600')}>${usdBalance}</div>}
-        </InUSD>
+      <div className="text-base font-medium text-gray-800">
+        <Money smallFractionFont={false}>{balance}</Money> <span>{getAssetSymbol(metadata)}</span>
       </div>
     ),
     [assetSlug, metadata]
```

Comment:
React Hook useCallback has an unnecessary dependency: 'assetSlug'. Either exclude it or remove the dependency array